package.json

```diff
@@ -76,6 +76,7 @@
     "@react-native-community/netinfo": "^5.5.1",
     "@react-native-community/push-notification-ios": "^1.2.0",
     "@react-navigation/core": "^3.5.1",
+    "@react-navigation/stack": "^5.9.0",
```

Comment:
no breaking changes here?

Reply:
i think no, as @MathiasMoralesDixtra already used 5.x version for the web navigation
i just aligned versions because app used both 4.x and 5.x